flink-yarn/src/main/java/org/apache/flink/yarn/YarnTaskManagerRunner.java

```diff
@@ -126,12 +124,6 @@ public static void runYarnTaskManager(String[] args, final Class<? extends YarnT
 				hadoopConfiguration.set(CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION, "kerberos");
```

Comment:
The block of code labeled 'integration test' seems dangerous.  It appears to override the Hadoop configuration whenever there's a keytab.    Since we're touching this code, it seems like the right time to remove or to defend this 'test' code.

Reply:
@EronWright , I've created a PR for [FLINK-8390|https://issues.apache.org/jira/browse/FLINK-8390] to address the integration test code issue. Could you please take a look at that as well?